src/data/faq.json

```
@@ -800,71 +800,11 @@
                                 "anchor": "val_service_basics",
                                 "active": false,
                                 "textblock": [
+                                    "<b>UPDATE  April 12, 2022</b><br/> The Europe-wide requirements for online verification services have already been in place since October 2021. In Germany, <b>no provider of an online verification service has been approved</b>. With version 2.15, however, the CWA is ready to support corresponding verification services.<hr/>",

```

```suggestion
                                    "<b>UPDATE April 12, 2022</b><br/> The Europe-wide requirements for online verification services have already been in place since October 2021. In Germany, <b>no provider of an online verification service has been approved</b>. With version 2.15, however, the CWA is ready to support corresponding verification services.<hr/>",
```

Remove double white space.